Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-alpine
+FROM openjdk:8
```

Comment:
The byte-size of that image seems to be about 5x larger than the alpine one. Did you find any issue using alpine?

Reply:
It is 9 month old and has an outdated jdk 1.8.0_212 installed. The current version is 1.8.0_242.

Reply:
Could you maybe replace it with an image which is up to date with a smaller size? It doesn't need to be alpine. In case you're using a different distribution, kindly also update the postgres & redis base-images accordingly.

Reply:
Switched to openjdk:8-slim as it is half as big as the standard openjdk:8. Also switched to redis:latest that is also based on Debian 10 (Buster). Postgres was already using Debian as a base image.